src/govuk/components/pagination/pagination.yaml

```diff
@@ -0,0 +1,169 @@
+params:
+- name: previous
+  type: object
+  required: false
+  description: A link to the previous page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The previous page link text. Defaults to 'Previous page', where 'page' is visually hidden.
+  - name: label
+    type: string
+    required: false
+    description: Optional label that goes underneath the previous page link providing further context for where it goes.
+  - name: href
+    type: string
+    required: true
+    description: The previous page URL.
+- name: next
+  type: object
+  required: false
+  description: A link to the next page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The next page link text. Defaults to 'Next page', where 'page' is visually hidden.
+  - name: label
+    type: string
+    required: false
+    description: Optional label that goes underneath the next page link providing further context for where it goes.
+  - name: href
+    type: string
+    required: true
+    description: The next page URL.
+- name: type
+  type: string
+  required: false
+  description: The type of navigation. Goes in the aria label for the encapsulating `nav` element. Defaults to "results".
```

Comment:
Will all our users know what 'encapsulating' means, or is there a plainer way of saying this?

Reply:
Feel free to ignore this as I've had 0 involvement in any pagination discussions up to this point, but I'm wondering if 'type' is the best name for this param 🤔 I'm guessing we don't want to use 'label' because it could be confused with the previous/next label options, but I'm not sure if it's clear that 'type' -> 'aria-label'. Feels like it might change the behaviour of the component, but I don't think it does?

Reply:
What about something like `navLabel` or `regionLabel`?

I also think the description could be simpler, perhaps something like:

```suggestion
  description: The label for the navigation landmark that wraps the pagination. Defaults to "results".
```

Reply:
I've followed Ollie's suggestion and changed the attribute to `landmarkLabel`.